wrappers/python/requirements.txt

```diff
@@ -1,3 +1,4 @@
 # grpc
 grpcio==1.23.0
 protobuf==3.9.2
+websocket-client==0.48.0
```

Comment:
Please remove it, as it is not needed for the wirepas-messaging packet. 

lets add a requirements inside the examples folder instead

Reply:
Do I need to add anything else than the requirements file? Packet creator can find it automatically?

Reply:
Drop a requirements.txt inside the examples/wnt with whatever extra package you need

Reply:
Can you check the latest code. I hope that it is fixed there.